support-frontend/assets/helpers/abTests/abtestDefinitions.ts

```diff
@@ -106,10 +106,13 @@ export const tests: Tests = {
 	threeTierCheckout: {
```

Comment:
I think we should give this test a new name as this'll make it easier to differentiate the results between test 1 and test 2, `threeTierCheckoutV2` would be fine

Reply:
Yes agree, otherwise I cant push it through into production either without affecting current test.

Reply:
Done - `threeTierCheckoutV2` setup with a `variantA` and `variantB` now/.